package.json

```diff
@@ -94,5 +94,8 @@
       "yarn run prettier --write",
       "git add"
     ]
+  },
```

Comment:
Is this intentional?

Reply:
yeah shouldn't be there in a `package.json`. if it's auto-formatting you could call `npm run lint` from the monorepo root (which should generate eslint + prettier config) and it hopefully will stop that.

Reply:
yeah this should be installed via the `travis.yml`?

Reply:
Im a bit confused about this. This file was not modified in this PR. What do I need to do?

Reply:
just remove this added dependency, it seems to have found it's way into the commit/PR even if you didn't intend it to?